Dockerfile

```diff
@@ -1,35 +1,82 @@
-FROM openshift/origin-release:golang-1.12 AS builder
+# Golang build container
+FROM golang:1.12.4
```

Comment:
why have anything other than what upstream has here, if we're doing the Dockerfile.ocp scheme anyways?

Reply:
That is the upstream Dockerfile. I am bringing it here to clear things up. ;)

Reply:
my mistake, that's exactly what I expected